sample_1.py

```diff
@@ -0,0 +1,16 @@
+def fib(n) :
+   if (n <= 0):
+       return 0
+   if (n >= 1):
+       return 1
+   if (n >= 2):
+       return 2
+   if (n >= 3):
+       f = n + 1 * [0]
+       fibo[1]=1 
+       sum = fibo[1] + fibo[0]
+       for i in range(2,n+1) :
+           fibo[i] = fibo[i-2] + fibo[i-1]
+           sum += fibo[i]
+   return sum
+
```

Comment:
otherwise, logic is correct but as suggested you can remove mutiple if else statements as it increases the line of code and also impacts the performance. This is a small example but as you will work on real life uses cases then over there you would definitely see the performance issues.

Reply:
Comments are added to the two sections of the code. Also, removed if statements as suggested.

Reply:
else:
            print('Input n should be of Integer type') 

You may also add else statement, but after adding it should not fail any of the test case basically please check that out

Reply:
Right, `else: print('Input n should be of Integer type')` has been added to explicitly show why input type failed.